### PR TITLE
V27snv v3sv UI updates

### PR DESCRIPTION
**Pull request by Bianca-Morris**
Includes popovers and styling fixes for V27 snv & v3 sv pipeline update. Also fixed some issues with facet extended description HTML files.

Note: Some of the HTML files added to the `src/encoded/docs/` folder are not in use as extended descriptions because those are currently only used for facets. These were added as JSX content, but the HTML files are left in case they are needed for facets at some point.

Compatible inserts are on cgaptest.

### CODE REVIEW

**Review by alexkb0009 — APPROVED** [review]
Looks great! (didn't run/test on local)

File: src/encoded/static/components/item-pages/StructuralVariantSampleView/SvDetailPanes.js [hunk 1]
```diff
@@ -139,38 +141,92 @@ function ConsequenceOfSVSection({ currentTranscript }) {
...
+                    <label htmlFor="sv-con-break1" className="mb-0" data-tip={null}>
                         Breakpoint 1:
+                        <QuickPopover popID="sv-con-break2-pop" className="p-0 ml-02 icon-sm" tooltip="Click for more information" title={breakpointPopoverTitle}>
+                            { breakpointPopoverContent }
+                        </QuickPopover>
                     </label>
                 </div>
-                <div className="col-12 col-xl-9" id="snv-con-break1">
+                <div className="col-12 col-xl-9" id="sv-con-break1">
                     { csq_variant_5_prime_location || fallbackElem }
```
Comment (drio18) [inline src/encoded/static/components/item-pages/StructuralVariantSampleView/SvDetailPanes.js:167]: Unless it's happening automatically here, could we remove the underscores from this field (e.g "5_prime_UTR" to "5 prime UTR) and for the other breakpoint location as indicated below?

File: src/encoded/static/components/item-pages/StructuralVariantSampleView/SvDetailPanes.js [hunk 2]
```diff
@@ -139,38 +141,92 @@ function ConsequenceOfSVSection({ currentTranscript }) {
...
+                    <label htmlFor="sv-con-break2" className="mb-0" data-tip={null}>
                         Breakpoint 2:
+                        <QuickPopover popID="sv-con-break2-pop" className="p-0 ml-02 icon-sm" tooltip="Click for more information" title={breakpointPopoverTitle}>
+                            { breakpointPopoverContent }
+                        </QuickPopover>
                     </label>
                 </div>
-                <div className="col-12 col-xl-9" id="snv-con-break2">
+                <div className="col-12 col-xl-9" id="sv-con-break2">
                     { csq_variant_3_prime_location || fallbackElem }
```
Comment (drio18) [inline src/encoded/static/components/item-pages/StructuralVariantSampleView/SvDetailPanes.js:180]: Underscores as above here as well.

**Review by drio18 — APPROVED** [review]
Everything here looks good. Both comments provided are for 1 small request that I don't think is included, but I could be wrong. Thanks for fixing all the HTML as well; I'm surprised it seemed to work thus far (or at least no one complained).

**Pull request merged**